airbyte-integrations/connectors/source-salesforce/unit_tests/integration/test_bulk_stream.py

```diff
@@ -8,23 +8,29 @@
 
 import freezegun
 from airbyte_cdk.test.mock_http import HttpMocker, HttpRequest, HttpResponse
-from airbyte_protocol.models import SyncMode
+from airbyte_protocol.models import AirbyteStreamStatus, SyncMode
 from config_builder import ConfigBuilder
+from integration.test_rest_stream import create_http_request as create_standard_http_request
+from integration.test_rest_stream import create_http_response as create_standard_http_response
 from integration.utils import create_base_url, given_authentication, given_stream, read
 from salesforce_describe_response_builder import SalesforceDescribeResponseBuilder
-from salesforce_job_response_builder import SalesforceJobResponseBuilder
+from salesforce_job_response_builder import JobCreateResponseBuilder, JobInfoResponseBuilder
 from source_salesforce.streams import LOOKBACK_SECONDS
 
 _A_FIELD_NAME = "a_field"
 _ACCESS_TOKEN = "an_access_token"
 _CLIENT_ID = "a_client_id"
 _CLIENT_SECRET = "a_client_secret"
 _CURSOR_FIELD = "SystemModstamp"
+_INCREMENTAL_FIELDS = [_A_FIELD_NAME, _CURSOR_FIELD]
+_INCREMENTAL_SCHEMA_BUILDER = SalesforceDescribeResponseBuilder().field(_A_FIELD_NAME).field(_CURSOR_FIELD, "datetime")  # re-using same fields as _INCREMENTAL_FIELDS
 _INSTANCE_URL = "https://instance.salesforce.com"
 _JOB_ID = "a-job-id"
 _LOOKBACK_WINDOW = timedelta(seconds=LOOKBACK_SECONDS)
 _NOW = datetime.now(timezone.utc)
 _REFRESH_TOKEN = "a_refresh_token"
+_RETRYABLE_RESPONSE = HttpResponse("{}", 420)  # TODO: document what the body actually is on 420 errors
```

Comment:
Nit: METHOD_FAILURE/ENHANCE_YOUR_CALM constant just for readability?

Reply:
TIL: 420 isn't even in the `http.HTTPStatus` from python hahahah I'll just extract 420 in a constant

Reply:
Yeah, I know. It was interesting to read about it, and I tried to get it from HTTPStatus too.